translator/source_code/icone_translator.py

```diff
@@ -21,8 +22,8 @@ def wzdx_creator(messages, info=None, unsupported_message_callback=None):
         return None
    #verify info obj 
     if not info:
-        info=initialize_info()
-    if not validate_info(info):
+        info = translator_shared_library.initialize_info()
+    if not translator_shared_library.validate_info(info):
```

Comment:
We're checking for nulls and validating up here which is great. Let's move the check for message incidents below (~line 64) up to here as well. That will prevent all this code being called if we have bad data in the first place.

Reply:
should be fixed